storage/rocksdb/clone/common.cc

```diff
@@ -101,6 +101,9 @@ void move_temp_dir_to_destination(const std::string &temp,
                              temp.c_str());
   }
 
+  myrocks::rocksdb_file_checksums =
```

Comment:
This does not appear to be the best place for this. Why not in `fixup_on_startup` in the same file?

Reply:
Moved it to the fixup_on_startup before renaming temp datadir to datadir.

Reply:
Tested in test server. In the restart during clone, it triggers the checksum verification.

> 2024-09-04T14:05:04.364182-07:00 0 [Note] [MY-011071] [Server] Plugin rocksdb reported: 'Verifying file checksums...'
2024-09-04T14:05:04.364244-07:00 0 [Note] [MY-011071] [Server] Plugin rocksdb reported: '...done'

Restart the recipient instance after clone finished, no checksum verification is triggered cause there is no related log reported by plugin rocksdb.